cea/utilities/solar_equations.py

```diff
@@ -132,13 +134,13 @@ def calc_sunrise(sunrise, Yearsimul, longitude, latitude):
     return sunrise
 
 
-def declination_degree(when, TY):
+def declination_degree(day_date, TY):
     """The declination of the sun is the angle between Earth's equatorial plane and a line
     between the Earth and the sun. It varies between 23.45 degrees and -23.45 degrees,
     hitting zero on the equinoxes and peaking on the solstices.
     Parameters
     ----------
-    when : datetime.datetime
+    day_date : datetime.datetime
```

Comment:
maybe we can check the parameter type (i think it might just be an int, the `day`?)